### ISSUE TITLE
Discord has been updated, does not work anymore

### ISSUE THREAD

**Issue by tadeokondrak** [NixOS/nixpkgs#54003]
## Issue description

Discord has been updated, does not work anymore.

Update probably needs to be backported to stable as well as the git (?)

### Steps to reproduce

`nix-shell -p discord --run discord`
 
```
[Modules] Checking for host updates.
[Updates] You are out of date!
[Modules] Host update is available. Manual update required!
```
![](https://hahay.es/Z90ph)

## Technical details

 - system: `"x86_64-linux"`
 - host os: `Linux 4.19.13, NixOS, 19.03pre165037.eebd1a92637 (Koi)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.1.3`
 - channels(root): `"nixos-19.03pre166308.626233eee6e"`
 - channels(tdeo): `"nixos-19.03pre165037.eebd1a92637"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

**Comment by tadeokondrak** [comment]
Turns out the change has already been made in the github to master (I only searched issues/pull requests), but I think it still needs to updated on stable.

**Comment by worldofpeace** [comment]
You're correct @tadeokondrak all updates to discord should be backported to stable because of this update block.

Though it would be really nice if we could report to discord that this is causing us constant problems and if there's a way we can disable this behavior.

**Comment by samueldr** [comment]
If upstream (discord) is contacted about that; it may be good to figure out a good contact point as a distro. I would ask if it would be possible to at least add a grace delay, like a couple days to a week, so channels and end-users can update without much worry. Additionally, a contact point which can point out technical changes that could affect us would be wonderful.

**Comment by worldofpeace** [comment]
It's also been considered if we can't achieve this ideal we could put this in a community maintained overlay that is automatically updated.

**Comment by MP2E** [comment]
Over the past couple of weeks I've tried getting a dialogue going about this but have not been successful so far, however I did learn that there's a discord development channel on discord itself that I can try, so I will do that. I think that a community maintained overlay that is automatically updated would be a great idea. I'm not entirely sure what would be needed to do that but I'm interested

**Comment by jcrben** [comment]
Couple ways to get a hold of them listed at https://discordapp.com/developers/docs/intro:
* https://github.com/discordapp/discord-api-docs/issues
* https://discordapp.com/invite/discord-api

**Comment by MP2E** [comment]
Thank you! Somehow missed that :)

**Comment by jonringer** [comment]
I used discord last night, I don't think this is relevant anymore. Please open a new ticket if this persists

**Comment by con-f-use** [comment]
And again, discord refuses to start without latest update.

**Comment by con-f-use** [comment]
Just for reference:

![image](https://user-images.githubusercontent.com/11145016/101259920-4889a100-372c-11eb-83b0-406cb768d743.png)

**Comment by jonringer** [comment]
this has been fixed in master and 20.09

you can install using:
```
nix-env -f https://github.com/NixOS/nixpkgs/archive/release-20.09.tar.gz -iA discord
```
or use in nix-shell:
```
NIX_PATH=nixpkgs=https://github.com/NixOS/nixpkgs/archive/release-20.09.tar.gz nix-shell -p discord --run Discord
```

**Comment by con-f-use** [comment]
Yeah, thanks, I'm aware. I just wanted a screenshot of the actual error in this issue, for reference so people can link it when troubleshooting.
